tests/manual-test-cases/Group5-Functional-Tests/5-29-Opaque-Network.robot

```diff
@@ -0,0 +1,121 @@
+    Close Connection
+
+    Log To Console  Set environment variables up for GOVC
+    Set Environment Variable  TEST_URL_ARRAY  ${vc-ip}
+    Set Environment Variable  TEST_URL  ${vc-ip}
+    Set Environment Variable  TEST_USERNAME  Administrator@vsphere.local
+    Set Environment Variable  TEST_PASSWORD  Admin\!23
+    Set Environment Variable  TEST_DATASTORE  vdnetSharedStorage
+    Set Environment Variable  TEST_DATACENTER  /os-test-dc
+    Set Environment Variable  TEST_RESOURCE  os-compute-cluster-1
+    Set Environment Variable  TEST_TIMEOUT  30m
+
+    Set Environment Variable  GOVC_INSECURE  1
+    Set Environment Variable  GOVC_URL  ${vc-ip}
+    Set Environment Variable  GOVC_USERNAME  Administrator@vsphere.local
+    Set Environment Variable  GOVC_PASSWORD  Admin\!23
+
+    Set Environment Variable  NSXT_MANAGER_URI  ${nsxt-mgr-ip}
+    Set Suite Variable  ${NIMBUS_POD}  ${pod}
+    Set Suite Variable  ${testrunid}  ${TESTRUNID}
+
+    OperatingSystem.Remove File  /tmp/vic-vdnet-nsxt.json
+
+Vdnet NSXT Topology Cleanup
+    [Timeout]    180 minutes
+    [Arguments]    ${pod_name}  ${testrunid}
+    Open Connection  %{NIMBUS_GW}
+    Wait Until Keyword Succeeds  10 min  30 sec  Login  %{NIMBUS_USER}  %{NIMBUS_PASSWORD}
+    Execute Command  ${NIMBUS_LOCATION} nimbus-ctl --nimbus=${pod_name} kill *${testrunid}*
+    Execute Command  ${NIMBUS_LOCATION} nimbus-ctl --nimbus=${pod_name} kill *isolated-06-gw
+    Close Connection
+
+*** Test Cases ***
+Basic VIC Tests with NSXT Topology
+    Log To Console  \nStarting test...
+    ${bridge_ls_name}=  Evaluate  'vic-bridge-ls'
+    ${result}=  Create Nsxt Logical Switch  %{NSXT_MANAGER_URI}  ${nsxt_username}  ${nsxt_password}  ${bridge_ls_name}
+    Should Not Be Equal    ${result}    null
+    ${container_ls_name}=  Evaluate  'vic-container-ls'
+    ${result}=  Create Nsxt Logical Switch  %{NSXT_MANAGER_URI}  ${nsxt_username}  ${nsxt_password}  ${container_ls_name}
+    Should Not Be Equal    ${result}    null
+    Set Environment Variable  BRIDGE_NETWORK  ${bridge_ls_name}
+    Set Environment Variable  CONTAINER_NETWORK  ${container_ls_name}
+    #Wait for 5 mins to make sure the logical switches is synchronized with vCenter
+    Sleep  300
+
+    Install VIC Appliance To Test Server  additional-args=--container-network-firewall=%{CONTAINER_NETWORK}:open --container-network-ip-range %{CONTAINER_NETWORK}:50.0.9.100-50.0.9.200 --container-network-gateway %{CONTAINER_NETWORK}:50.0.9.1/24
+    Run Regression Tests
+
+    #Validate containers' connection when using container network
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -it -d --net=public --name first ${busybox}
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+
+    ${ip}=  Get Container IP  %{VCH-PARAMS}  first  public
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --net=public ${debian} ping -c1 ${ip}
```

Comment:
We should use variables to refer to images, as Yang has done, so that we can pull them from different registries based on where the tests are running. This helps prevent the availability of third-party systems from affecting the stability of our CI system.